wasm/examples/nodejs/javascript/general/derivation.js

```diff
@@ -0,0 +1,67 @@
+const kaspa = require('../../../../nodejs/kaspa');
+const {
+    Mnemonic,
+    XPrv,
+    DerivationPath,
+    PublicKey,
+    NetworkType,
+} = kaspa;
+
+kaspa.initConsolePanicHook();
+
+(async () => {
+
+    //const mnemonic = Mnemonic.random();
+    const mnemonic = new Mnemonic("hunt bitter praise lift buyer topic crane leopard uniform network inquiry over grain pass match crush marine strike doll relax fortune trumpet sunny silk")
+    console.log("mnemonic:", mnemonic);
+    const seed = mnemonic.toSeed();
+    console.log("seed:", seed);
+
+    // kaspa
+    let xPrv = new XPrv(seed);
+    // derive full path upto second address of receive wallet
+    let pubkey1 = xPrv.derivePath("m/44'/111111'/0'/0/1").toXPub().publicKey();
+    console.log("address", pubkey1.toAddress(NetworkType.Mainnet));
+
+    // create receive wallet
+    let receiveWalletXPub = xPrv.derivePath("m/44'/111111'/0'/0").toXPub();
+    // derive receive wallet for second address
+    let pubkey2 = receiveWalletXPub.deriveChild(1, false).publicKey();
+    console.log("address", pubkey2.toAddress(NetworkType.Mainnet));
+
+    // create change wallet
+    let changeWalletXPub = xPrv.derivePath("m/44'/111111'/0'/1").toXPub();
+    // derive change wallet for first address
+    let pubkey3 = changeWalletXPub.deriveChild(0, false).publicKey();
```

Comment:
```suggestion
    let pubkey3 = changeWalletXPub.deriveChild(0, false).toPublicKey();
```

Reply:
Fixed in https://github.com/aspectron/rusty-kaspa/commit/a0d677aa7f581d13324baba4df4eaddd2caefc21